README.md

```diff
@@ -10,7 +10,7 @@ RcppDeepState is a fuzz testing library made as a composition of three tools: Rc
 -   **fail_ci_if_error** (default value: `false`) - Specify if CI pipeline should fail when RcppDeepState finds errors;
 -   **location** (default value: `/`) - Relative path under `$GITHUB_WORKSPACE` that contains the package that needs to be analyzed. Default uses the `/` location relative to `$GITHUB_WORKSPACE`, that is `$GITHUB_WORKSPACE`;
 -   **seed** (default value: `-1`) - control the randomness of the inputs generated in the fuzzing phase;
--   **time_limit** (default value: `5`) - Fuzzing phase's duration in seconds;
+-   **time_limit_seconds** (default value: `5`) - Fuzzing phase's duration in seconds;
```

Comment:
this is the duration for every input or for every function? maybe could be further clarified by changing the name to something like `max_seconds_per_function` ?

Reply:
I agree with you; this makes it clearer for the end user the purpose of the parameter. 
Updated with commit aba09b7